{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,47 +1,23 @@
-# ref https://github.com/tebeka/pythonwise/blob/master/docker-miniconda/Dockerfile
-FROM python:3.10-slim-buster
+# # SPDX-License-Identifier: MIT
+# # FileType: SOURCE
+# # FileCopyrightText: 2024, FernLab at GFZ Potsdam
 
-# System packages
-RUN apt-get update -y && \
-    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
-    apt-get install -y -q dialog apt-utils && \
-    apt-get install libudunits2-dev build-essential bash-completion automake -y && \
-    apt-get install cmake zlib1g-dev libpng-dev libjpeg-dev -y  && \
-    apt-get install wget vim zip unzip curl git git-lfs jq apt-rdepends -y && \
-    apt-get install strace bzip2 cron make gcc -y && git-lfs install
 
-# Install mambaforge
-RUN curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh
-RUN bash Mambaforge-$(uname)-$(uname -m).sh -b -p /mambaforge
-RUN rm Mambaforge-$(uname)-$(uname -m).sh
-ENV PATH=/mambaforge/bin:${PATH}
+# # Container for building the environment
+FROM condaforge/miniforge3:latest
+RUN apt-get update -y
 
-# Python packages from conda
-RUN mamba install -y python=3.9
-RUN mamba install -y pip
-RUN mamba init bash
-
-# copy some needed stuff to /root
+# Create mamba environment
 COPY /tests/CI_docker/context/environment_{{ cookiecutter.project_slug }}.yml /root/
 RUN mamba env create -f /root/environment_{{ cookiecutter.project_slug }}.yml
 
-# Make RUN commands use the new environment:
+# Activate the environment
 SHELL ["conda", "run", "-n", "{{ cookiecutter.project_slug }}", "/bin/bash", "-c"]
 
-# Allow statements and log messages to immediately appear in the Knative logs
-ENV PYTHONUNBUFFERED True
-
-# Install {{ cookiecutter.project_slug }}
-ADD . /{{ cookiecutter.project_slug }}/
+# Copy source code
```

Comment:
We only create a docker image with the correct environment. The package should be installed.

Reply:
Good point!